### PR TITLE
Adding Delete Tape and Get Tapes support

### PR DESCRIPTION
**Pull request by rpmoore**

### CODE REVIEW

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/DeleteTape.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package com.spectralogic.ds3cli.command;
```
Comment (DenverM80) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/DeleteTape.java]: Copyright?

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/json/GetTapesView.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.spectralogic.ds3cli.View;
+import com.spectralogic.ds3cli.models.GetTapesResult;
+import com.spectralogic.ds3cli.util.JsonMapper;
+import com.spectralogic.ds3client.models.tape.Tapes;
+
+public class GetTapesView implements View<GetTapesResult> {
+    @Override
+    public String render(final GetTapesResult obj) throws JsonProcessingException {
+       final Tapes result = obj.getTapes();
```
Comment (RachelTucker) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/json/GetTapesView.java]: Inconsistent indentation

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/json/GetTapesView.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        final CommonJsonView view = CommonJsonView.newView(CommonJsonView.Status.OK);
+
+        if( (result == null) || (null == result.getTapes()) ){
+            view.message("You do not have any buckets");
```
Comment (DenverM80) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/json/GetTapesView.java]: "You do not have any tapes"

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetTapes.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        return this;
+    }
+
+    @Override
+    public GetTapesResult call() throws Exception {try {
+            final GetTapesResponse response = getClient().getTapes(new GetTapesRequest());
+
+            return new GetTapesResult(response.getTapes());
+        } catch (final FailedRequestException e) {
+            throw new CommandException("Failed Get Service", e);
```
Comment (DenverM80) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetTapes.java]: Failed Get Tapes..?

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/json/GetTapesView.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+import com.spectralogic.ds3client.models.tape.Tapes;
+
+public class GetTapesView implements View<GetTapesResult> {
+    @Override
+    public String render(final GetTapesResult obj) throws JsonProcessingException {
+        final Tapes result = obj.getTapes();
+        final CommonJsonView view = CommonJsonView.newView(CommonJsonView.Status.OK);
+
+        if( (result == null) || (null == result.getTapes()) ){
+            view.message("You do not have any buckets");
```
Comment (DenverM80) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/json/GetTapesView.java]: *tapes

**Pull request merged**